<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[==DB Connection Successful==]]>
    </line>
    <line>
      <![CDATA[==db closed==]]>
    </line>
  </reporter-output>
  <suite started-at="2022-09-30T11:12:00 IST" name="Default suite" finished-at="2022-09-30T11:12:01 IST" duration-ms="1502">
    <groups>
    </groups>
    <test started-at="2022-09-30T11:12:00 IST" name="Default test" finished-at="2022-09-30T11:12:01 IST" duration-ms="1502">
      <class name="EndToEndScenario.CreateProjectAndVerifyInDB">
        <test-method is-config="true" signature="bsConfig()[pri:0, instance:EndToEndScenario.CreateProjectAndVerifyInDB@7205765b]" started-at="2022-09-30T11:11:59 IST" name="bsConfig" finished-at="2022-09-30T11:12:00 IST" duration-ms="617" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[==DB Connection Successful==]]>
            </line>
          </reporter-output>
        </test-method> <!-- bsConfig -->
        <test-method signature="createProject()[pri:0, instance:EndToEndScenario.CreateProjectAndVerifyInDB@7205765b]" started-at="2022-09-30T11:12:00 IST" name="createProject" finished-at="2022-09-30T11:12:01 IST" duration-ms="1474" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [TY_PROJ_3003] but found [TY_PROJ_807]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [TY_PROJ_3003] but found [TY_PROJ_807]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at EndToEndScenario.CreateProjectAndVerifyInDB.createProject(CreateProjectAndVerifyInDB.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createProject -->
        <test-method is-config="true" signature="asConfig()[pri:0, instance:EndToEndScenario.CreateProjectAndVerifyInDB@7205765b]" started-at="2022-09-30T11:12:01 IST" name="asConfig" finished-at="2022-09-30T11:12:01 IST" duration-ms="5" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[==db closed==]]>
            </line>
          </reporter-output>
        </test-method> <!-- asConfig -->
      </class> <!-- EndToEndScenario.CreateProjectAndVerifyInDB -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
